playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/kubeflow_tfbenchmarks.yaml

```diff
@@ -133,76 +133,76 @@
       shell: |
         set -o pipefail
         set -ex
-        kubectl -n kf-tfbenchmarks create -f "{{ ansible_user_dir }}/workspace/kf-tfbenchmarks/{{ item }}"
-        sleep 20
-        kubectl -n kf-tfbenchmarks get tfjobs -o wide
-        kubectl -n kf-tfbenchmarks get pods -o wide
+        # We only test the typical case case4, otherwise, it will take very long time.
+        # cases="case1 case2 case3 case4"
+        cases="case4"
```

Comment:
Or maybe we should use a variable to set this instead hard coding?

Reply:
done

Reply:
Thanks. And it's better to change the two other places, see comments.

Reply:
also changed